plugins/woocommerce/src/Internal/Admin/Settings/PaymentsController.php

```diff
@@ -170,7 +170,6 @@ public function add_allowed_promo_notes( $promo_notes = array() ): array {
 				'Failed to get payment providers: ' . $e->getMessage(),
 				array(
 					'source' => 'settings-payments',
-					'error'  => $e,
```

Comment:
Don't include the entire error object to about stuffing the error logs with too much data.